### PR TITLE
Initial Import API

### PR DESCRIPTION
**Pull request by joratrifork**

### CODE REVIEW

File: openapi/schemas/ImportRegistrationResponse.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+items:
+  required: ["registrationId", "status"]
+  properties:
+    registrationId:
+      type: "string"
+    status:
+      type: "string"
+      description: "Describes the status of the registration (SUCCESS, PENDING, FAILED)"
+    message:
+      description: "When the status = FAILED, this will provide a description of the problem"
```
Comment (hmmtrifork) [inline openapi/schemas/ImportRegistrationResponse.yaml:11]: remove "the" before status

File: openapi/trireg2.yaml [hunk 2]
```diff
@@ -2,18 +2,20 @@ openapi: "3.0.3"
 info:
   title: "TriReg2 API"
   description: "An OpenAPI specification for time registrations"
-  version: "0.0.5"
+  version: "0.0.6"
 servers:
-  - url: "http://0.0.0.0:8080"
+- url: "http://0.0.0.0:8080"
```
Comment (hmmtrifork) [inline openapi/trireg2.yaml:7]: is this indentation correct?

File: openapi/trireg2.yaml [hunk 3]
```diff
@@ -25,5 +27,5 @@ components:
     TimeRegistrationResponse:
       $ref: "./schemas/TimeRegistrationResponse.yaml"
 security:
-  - openId:
-      - openid
\ No newline at end of file
+- openId:
```
Comment (hmmtrifork) [inline openapi/trireg2.yaml:30]: is this indentation correct?

**Review by hmmtrifork — APPROVED** [review]
Looks good to me.
There are lint warnings, should those be fixed as well before merging?

File: openapi/trireg2.yaml (hunk 2 above)
Comment (joratrifork) [reply openapi/trireg2.yaml:7]: Yes ( https://stackoverflow.com/a/71968274 ) for yaml sequences the indentation level doesn't matter. It looks like the VSC extension I have for yaml formatting prefers this style.

**Pull request merged**